plugins/modules/kubevirt_vm.py

```diff
@@ -61,8 +61,20 @@
   running:
     description:
     - Specify whether the C(VirtualMachine) should be running or not.
+    - Mutually exclusive with O(run_strategy).
```

Comment:
Is this enforced anywhere within this module?

Reply:
Yes, see L425